account_invoice_partner_no_autofollow/models/account_move.py

```diff
@@ -0,0 +1,29 @@
+from odoo import api, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    def message_subscribe(self, partner_ids=None, channel_ids=None, subtype_ids=None):
+        if self.env.context.get("no_auto_follow"):
+            return False
+        return super(AccountMove, self).message_subscribe(
+            partner_ids, channel_ids, subtype_ids
+        )
+
+    @api.model_create_multi
+    def create(self, values):
+        result = super(AccountMove, self).create(values)
+        if self.env["ir.config_parameter"].sudo().get_param("account.no_auto_follow"):
```

Comment:
Why do unsubscribe partners here explicitly if this can be done via context?

Reply:
fixed